src/AnnotatedCommand.php

```diff
@@ -225,8 +225,9 @@ protected function setCommandArguments($commandInfo)
      */
     protected function checkUsesInputInterface($params)
     {
+        /** @var \ReflectionParameter $firstParam */
         $firstParam = reset($params);
-        return $firstParam instanceof InputInterface;
+        return $firstParam->getClass() && $firstParam->getClass()->implementsInterface(InputInterface::class);
```

Comment:
Is there a reason you don't like `instanceof`, though? It's an older language construct, but shorter than the suggested replacement. I am indifferent to `instanceof` vs `implementsInterface()` here.